packages/charts/package.json

```
@@ -23,7 +23,7 @@
     "recharts": "2.3.2"
   },
   "peerDependencies": {
-    "@ui5/webcomponents-react": "^1.1.0",
+    "@ui5/webcomponents-react": "^1.1.3",
```

```suggestion
    "@ui5/webcomponents-react": "^1.1.0",
```
not required, please revert